services/types.go

```diff
@@ -41,6 +41,12 @@ type Client interface {
 		*types.PartialBlockIdentifier,
 	) (*types.Block, error)
 
+	Transaction(
```

Comment:
Might be better to use `fetchTransaction` as method name.

Reply:
I kept it similar to `Block` function to keep consistency.